cristin-import/build.gradle

```diff
@@ -31,6 +32,7 @@ implementation libs.jackson.databind
     implementation libs.aws.sdk2.urlconnectionclient
     implementation libs.aws.sdk2.regions
 
+    implementation libs.typesafe.config
```

Comment:
Using lightbend config (https://github.com/lightbend/config) for enabling/disabling the online validation. 
I was between customizing the DoiConverter to accept a custom validation logic for running tests or to use the configuration file to enable online validation in production and to disable it during unit testing. I went for the option that created the least complex code. 
The price is that we have to load more stuff in the startup but this is a batch job and response time is not very important.